nextflow.config

```diff
@@ -94,10 +94,11 @@ params {
 
 // Container slug. Stable releases should specify release tag!
 // Developmental code should specify :dev
-process.container = 'nfcore/mhcquant:1.5.1'
+process.container = 'nfcore/mhcquant:1.6.0'
 
 // Load base.config by default for all pipelines
 includeConfig 'conf/base.config'
+includeConfig 'conf/test_full.config'
```

Comment:
```suggestion
```

Reply:
so that you can call the `test_full` profile, it needs to be added like:
`test_full { includeConfig 'conf/test_full.config' }` 
in the profiles section (see how this is done for the `test.config` in the line 126.